src/components/sbb-breadcrumb/sbb-breadcrumb.tsx

```diff
@@ -0,0 +1,110 @@
+import { Component, ComponentInterface, Element, h, Host, JSX, Prop, State } from '@stencil/core';
+import {
+  LinkProperties,
+  LinkTargetType,
+  resolveLinkOrStaticRenderVariables,
+  targetsNewWindow,
+} from '../../global/interfaces/link-button-properties';
+import { i18nTargetOpensInNewWindow } from '../../global/i18n';
+import {
+  actionElementHandlerAspect,
+  createNamedSlotState,
+  documentLanguage,
+  HandlerRepository,
+  languageChangeHandlerAspect,
+  namedSlotChangeHandlerAspect,
+} from '../../global/helpers';
+
+/**
+ * @slot unnamed - Use this to slot the breadcrumb's text.
+ * @slot icon - Use this to display an icon as breadcrumb.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-breadcrumb.scss',
+  tag: 'sbb-breadcrumb',
+})
+export class SbbBreadcrumb implements ComponentInterface, LinkProperties {
+  /** The href value you want to link to. */
+  @Prop() public href: string | undefined;
+
+  /** Where to display the linked URL. */
+  @Prop() public target?: LinkTargetType | string | undefined;
+
+  /** The relationship of the linked URL as space-separated link types. */
+  @Prop() public rel?: string | undefined;
+
+  /** Whether the browser will show the download dialog on click. */
+  @Prop() public download?: boolean;
+
+  /**
+   * The icon name we want to use, choose from the small icon variants
+   * from the ui-icons category from here
+   * https://lyne.sbb.ch/tokens/icons/.
+   * Inline variant doesn't support icons.
+   */
```

Comment:
```suggestion
  /**
   * The name of the icon, choose from the small icon variants
   * from the ui-icons category from here
   * https://icons.app.sbb.ch.
   */
```

Reply:
Fixed in 94d8eadd8072b7786a8d757bcb0acabd932179e0